File: src/main/java/net/hectus/neobb/turn/default_game/block/TSoulSand.java
package net.hectus.neobb.turn.default_game.block;

import net.hectus.neobb.player.NeoPlayer;
import net.hectus.neobb.turn.default_game.attributes.clazz.SupernaturalClazz;
import net.hectus.neobb.turn.default_game.attributes.function.AttackFunction;
import org.bukkit.block.Block;

public class TSoulSand extends BlockTurn implements AttackFunction, SupernaturalClazz {
    public TSoulSand(NeoPlayer player) { super(player); }
    public TSoulSand(Block data, NeoPlayer player) { super(data, player); }

    @Override
    public int cost() {
        return 3;
    }
}

File: src/main/kotlin/net/hectus/neobb/turn/default_game/block/TSoulSand.kt
package net.hectus.neobb.turn.default_game.block

import com.marcpg.libpg.storing.Cord
import net.hectus.neobb.player.NeoPlayer
import net.hectus.neobb.turn.default_game.attribute.clazz.SupernaturalClazz
import net.hectus.neobb.turn.default_game.attribute.function.AttackFunction
import org.bukkit.block.Block

class TSoulSand(data: Block?, cord: Cord?, player: NeoPlayer?) : BlockTurn(data, cord, player), AttackFunction, SupernaturalClazz {
    override val cost: Int = 3
}